packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/components/TraitGridFilters.tsx

```diff
@@ -41,19 +42,21 @@ const Wrapper = styled.div`
 
 const ActiveTraitFilter = styled.div`
   align-items: center;
-  background: ${colors.blue000};
-  border: 1px solid ${colors.blue200};
-  border-radius: 8px;
+  background: #fafbff;
```

Comment:
In design, but doesn't have a color in constellation... would kinda be like grey(-100) or something

Reply:
design needs to change then